practice-typescript/src/scripts/controllers/user.ts

```diff
@@ -42,6 +43,12 @@ class UserController {
   handleGetById = async (id: string): Promise<UserModel> => {
     return await this.service.getById(id);
   };
+
+  handleSearchByName = async (email: string): Promise<any> => {
```

Comment:
Search by name but pass the email param? And please check the return type of the function